src/loader/RevolvingDot.tsx

```diff
@@ -5,13 +5,14 @@ import { BaseProps, DEFAULT_COLOR, DEFAULT_WAI_ARIA_ATTRIBUTE } from '../type'
 interface RevolvingDotProps extends BaseProps {
   radius?: string | number
   secondaryColor?: string
+  strokeWidth?: string | number
```

Comment:
Rethink about the props type for the spinner. Number will make more sense and we can limit it to supported range.

Reply:
I agree, we can keep it a number.
But I think we should not have a limit to it, people may want to have granular control for the component.

In this case they can mix and match `radius` and `strokeWidth` according to their preference